backend/scripts/test.sh

```
@@ -18,28 +18,80 @@ export TESTING=true
+        result = conn.execute(text('SELECT 1'))
+        print('✅ Database connection successful')
+except Exception as e:
+    print(f'❌ Database connection failed: {e}')
+    sys.exit(1)
+" || {
+    echo "❌ Database connectivity check failed"
+    exit 1
+}
+
 # Set default title for coverage report
 TITLE="${@:-coverage}"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)